libtcmu_log.c

```diff
@@ -395,7 +400,7 @@ static int output_to_fd(int pri, const char *timestamp,
 	char *buf, *msg;
 	int count, ret, written = 0, r, pid = 0;
 
-	if (fd < 0)
+	if (fd ==  -1)
```

Comment:
Drop the extra space between "==" and -1.

Reply:
Done